sormas-app/app/src/main/java/de/symeda/sormas/app/component/controls/ControlDateField.java

```diff
@@ -68,6 +68,8 @@ public class ControlDateField extends ControlPropertyEditField<Date> {
 	private int allowedDaysInFuture;
 	private Date cachedTime;
 
+	private boolean simpleDate;
```

Comment:
```suggestion
	private boolean dateWithoutTime;
```

=> more self-explanatory because it's not directly obvious what a "simple" date is supposed to be